src/hooks/portefolje/use-sync-state-med-url.ts

```diff
@@ -18,7 +18,9 @@ export function useSyncStateMedUrl() {
     useEffect(() => {
         if (side) {
             const parsed = queryString.parse(window.location.search);
+            // @ts-ignore
             parsed.side = side;
+            // @ts-ignore
```

Comment:
Er det mulig å legge til bedre typer på "side" for å unngå ts-ignore?

Reply:
Jeg har gått tilbake til type any på paginering i AppState (reducer.ts) da forsvinner type-sjekk feilene.